intuit.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" type="text/css" href="intuit.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <title>Intuit Page</title>
+  </head>
+  <body class="flex">
+    <header>
+      <div>
+        <a>Already have an Intuit account?</a><br />
+        <a
+          class="links_blue"
+          href="https://accounts.intuit.com/account-recovery.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D97b97818-aa3d-4a83-bf6c-63cc957cf016%26adobe_mc%3DMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1588629675%26ivid%3D97b97818-aa3d-4a83-bf6c-63cc957cf016"
+          target="_blank"
+          >I forgot my User ID or password</a
+        >
+      </div>
+      <div>
+        <button class="sign_in_btn" type="submit">
+          <i class="fas fa-lock"></i> Sign In
+        </button>
+      </div>
+    </header>
+    <main>
+      <div class="intuit_header">
+        <img
+          class="intuit"
+          src="https://www.bayleafdigital.com/wp-content/uploads/2018/11/logo-intuit-gray.png"
+          alt="intuit_logo"
+        />
+      </div>
+      <div>
+        <img
+          id="green_leaf"
+          src="https://digitalasset.intuit.com/IMAGE/A3OSvbxLf/logo-mint-footer.png"
+          alt="green_leaf"
+        />
+        <img
+          id="quickbook"
+          src="https://digitalasset.intuit.com/IMAGE/A7Aq9134m/logo-quickbooks.png"
+          alt=" quickbook"
+        />
+        <img
+          id="turbo_tax"
+          src="https://digitalasset.intuit.com/IMAGE/A4ldtKC3b/logo-turbotax.png"
+          alt="turbo_tax"
+        />
+      </div>
```

Comment:
You are using an image containing a logo as well as the text of the product but the text should not be in the image. [Link to the page](https://accounts.intuit.com/signup.html)
![Screenshot from 2020-05-09 15-26-01](https://user-images.githubusercontent.com/37747619/81470232-6d0ab280-9209-11ea-959d-bef14698c76f.png)